commands/Moderation/mute.js

```diff
@@ -0,0 +1,59 @@
+exports.run = async (client, msg, [user]) => {
+  const { muteRoleId, reportChannelId, titleURL, muteRoleName } = msg.guild.settings;
+  try {
+    const value = msg.mentions.members.first().roles.get(muteRoleId);
+    if (msg.mentions.members.first().roles.get(muteRoleId)) {
+      msg.mentions.members.first().removeRole(msg.guild.roles.find("name", muteRoleName));
```

Comment:
Thought you'll remove the `msg.mentions.members.first()` with the previous requested change, that's a lot of duplicated code.